### PR TITLE
changes for compiling brotli on old centos6 distro

### PR DESCRIPTION
**Pull request by ericmolle**
As requested by @slandelle, here is a PR for building on Centos6.

### CODE REVIEW

File: natives/pom.xml [hunk 1]
```diff
@@ -29,12 +29,25 @@ Copyright 2021, Aayush Atharva
     <packaging>pom</packaging>
     <modules>
         <module>linux-x86_64</module>
+        <module>linux-x86_64-centos6</module>
```
Comment (slandelle) [inline natives/pom.xml:32]: No, we shouldn't have a special module for centos6. Your suggested changes should target the existing linux-x86_64 module.

Also, the names of those modules match Netty's expected pattern wrt native libs.

**Review by slandelle — CHANGES_REQUESTED** [review]

File: natives/pom.xml (hunk 1 above)
Comment (ericmolle) [reply natives/pom.xml]: I've made changes, as suggested to replace existing linux-x86_64 by the centos build

**Comment by slandelle** [comment]
@hyperxpro Could you please review and if possible release? FIY, I've tested the generated binary on latest Amazon Linux and it works there as well (not only CentOS 6).

File: natives/pom.xml (hunk 1 above)
Comment (hyperxpro) [reply natives/pom.xml]: I think we should call it `linux-centos6-x86-64` to keep naming constant with architectures. WDYT @slandelle ?

Comment (slandelle) [reply natives/pom.xml]: No. The binary is generic enough to work fine on more modern distributions, just like Netty's own binaries.

**Comment by hyperxpro** [comment]
@slandelle Can you test this binary, please? https://github.com/hyperxpro/Brotli4j/suites/2619416175/artifacts/57397865

**Comment by slandelle** [comment]
Works as well :)

**Comment by hyperxpro** [comment]
@slandelle I'll be releasing by this weekend. Busy with something right now. Really sorry for the release delay. :(

**Comment by hyperxpro** [comment]
Additionally, can you guys help me in creating a release workflow? It'll be extremely handy for releasing builds.

**Comment by slandelle** [comment]
@hyperxpro Sorry, we won't be able to spend more time on this anytime soon. We've already spent a crazy amount of time building a CentOS6 Docker image shipping a modern cpp compiler required by brotli so that the binary would match for Netty's requirements.

We do have a similar story in our backlog for some internal JNI components, so that's something we'll tackle some day and we'll contribute here then.

Do you have any idea when you might be able to do a manual release?

**Comment by hyperxpro** [comment]
@slandelle No problem, I really appreciate you and your team's effort on this PR. I will be releasing a new version in the next 2 days.

**Comment by slandelle** [comment]
Thanks a lot.

**Comment by hyperxpro** [comment]
@slandelle Released!

**Comment by slandelle** [comment]
Awesome, thanks! I'll be watching for the maven central sync and then upgrade both Gatling and my PR against netty.

**Comment by hyperxpro** [comment]
Thanks a lot for your all efforts! :)

**Comment by slandelle** [comment]
Sadly, there's still an issue on netty's CI:

```
java.lang.UnsatisfiedLinkError: /tmp/com_aayushatharva_brotli4j_1036749389702/libbrotli.so: /usr/lib64/libstdc++.so.6: version `CXXABI_1.3.9' not found (required by /tmp/com_aayushatharva_brotli4j_1036749389702/libbrotli.so)
```

We'll investigate.

**Pull request merged**